### ISSUE TITLE
Issue with service php5-fpm reload on Ubuntu 14.04

### ISSUE THREAD

**Issue by SergK** [thias/puppet-php#55]
Hi,
I have an issue on:
```
Distributor ID:	Ubuntu
Description:	Ubuntu 14.04.1 LTS
Release:	14.04
Codename:	trusty
```
package:
```
ii  php5-fpm    5.5.9+dfsg-1ubuntu4.5      amd64    server-side, HTML-embedded scripting language (FPM-CGI binary)
```
It seems that customize service reload doesn't work correctly, I'd propose to put restart:
https://github.com/thias/puppet-php/blob/master/manifests/fpm/daemon.pp#L34
```
- restart   => "service ${fpm_service_name} reload",
+ restart   => "service ${fpm_service_name} restart",
```

```bash
root@slave-08:/etc/php5/fpm/pool.d# ps auxwwf |grep php
root     22795  0.0  0.0  10464   920 pts/2    S+   09:51   0:00                          \_ grep --color=auto php
root     21503  0.0  0.7 315644 16232 ?        Ss   09:49   0:00 php-fpm: master process (/etc/php5/fpm/php-fpm.conf)                    
www-data 21506  0.0  0.6 317748 12328 ?        S    09:49   0:00  \_ php-fpm: pool www                                                       
www-data 21507  0.0  0.6 317748 12328 ?        S    09:49   0:00  \_ php-fpm: pool www                                                       
www-data 21508  0.0  0.7 319548 16192 ?        S    09:49   0:00  \_ php-fpm: pool www                                                       
www-data 21509  0.0  0.7 319548 16040 ?        S    09:49   0:00  \_ php-fpm: pool www                                                       
www-data 21510  0.0  0.9 319812 19176 ?        S    09:49   0:00  \_ php-fpm: pool www                                                       
www-data 21801  0.0  0.5 317748 12184 ?        S    09:50   0:00  \_ php-fpm: pool www                                                       
root@slave-08:/etc/php5/fpm/pool.d# service php5-fpm reload
root@slave-08:/etc/php5/fpm/pool.d# ps auxwwf |grep php
root     22891  0.0  0.0  10464   916 pts/2    S+   09:51   0:00                          \_ grep --color=auto php
www-data 21506  0.0  0.7 319644 15156 ?        S    09:49   0:00 php-fpm: pool www                                                       
www-data 21507  0.0  0.7 319292 15024 ?        S    09:49   0:00 php-fpm: pool www                                                       
www-data 21508  0.0  0.8 319548 16408 ?        S    09:49   0:00 php-fpm: pool www                                                       
www-data 21509  0.0  0.7 319548 16040 ?        S    09:49   0:00 php-fpm: pool www                                                       
www-data 21510  0.0  0.9 319812 19592 ?        S    09:49   0:00 php-fpm: pool www                                                       
www-data 21801  0.0  0.7 319292 15500 ?        S    09:50   0:00 php-fpm: pool www                                            
``` 

Thank you

**Comment by gblair** [comment]
:+1: Or at least the option to specify whether to use `reload` or `restart`. We are having the same issue, due to [a languishing Ubuntu PHP-FPM bug](https://bugs.launchpad.net/ubuntu/+source/php5/+bug/1242376).

**Comment by thias** [comment]
This should be fixed in 910ffa17cf84707b66f94a3497374f9e8b3119c1 and will be included in the next release of the module.
